server/build/webpack.js

```diff
@@ -168,7 +169,16 @@ export default async function createCompiler (dir, { dev = false, quiet = false
               require.resolve('babel-plugin-module-resolver'),
               {
                 alias: {
-                  'styled-jsx/style': require.resolve('styled-jsx/style')
```

Comment:
Hey @timneutkens styled-jsx is not in the list now?
I think we'll have the `styled-jsx` problem again.